pom.xml

```diff
@@ -657,5 +657,24 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>kafka-0.10.0.1</id>
```

Comment:
This version is actually lower than kafka-0.10.dev where it was using kafka 0.10.2.0, is there a particular reason that you need to use 0.10.0.1?

Reply:
I can't remember the specific errors but when trying to consume from a 0.10.0.1 cluster it does not work.